drivers/cache/Kconfig

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2021 Carlo Caione <ccaione@baylibre.com>
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig EXTERNAL_CACHE
```

Comment:
Is this intentional to keep the EXTERNAL prefix here, while the directory path is just "cache"?

Reply:
I found it to be more descriptive but I'll change it to match the directory name.